src/imitation/scripts/config/tune.py

```diff
@@ -0,0 +1,45 @@
+"""Config files for hyperparameter tuning experiments.
+
+Hyperparameter experiments are intended to be defined in Python rather than
+via CLI. For example, a user should add a new
+`@tune_ex.named_config` to define a new hyperparameter experiment.
+
+Adding custom named configs is necessary because the CLI interface can't add
+search spaces to the config like `"seed": tune.grid_search([0, 1, 2, 3])`.
+"""
+import ray.tune as tune
+import sacred
+
+tune_ex = sacred.Experiment("tune")
+
+
+@tune_ex.config
+def config():
+  inner_experiment_name = "expert_demos"  # The experiment to tune
+  search_space = {
+    "named_configs": [],
+    "config_updates": {},
+  }  # `config` argument to `ray.tune.run(trainable, config)`
+
+
+# Debug named configs
+@tune_ex.named_config
+def debug_log_root():
+  search_space = {"config_updates": {"log_root": "/tmp"}}
```

Comment:
Consider a subdirectory of `/tmp` -- it'll be hard to keep track of stuff otherwise